### DISCUSSION TITLE
Solax Sensoren Werte MQTT bereitstellen

### DISCUSSION THREAD

**Discussion by westi2000** [wills106/homeassistant-solax-modbus#787] [General]
Hallo, ich würde gerne die Daten von den Sensoren aus HA über MQTT auf einen Brocker ablegen, um sie für weitere Geräte zu nutzen (Victron Cerbo GX über Note red). Hat jemand eine Idee wie das gehen könnte? Ich bin aber Newbie und brauche es einfach erklärt :)
Danke

**Comment by wrfz** [comment]
I think this question has little or nothing to do with Solax integration, but is more of a general HA topic.
There might be something for you here:
https://community.home-assistant.io/t/looking-for-a-sample-to-publish-an-entity-value-to-mqtt/504519
https://community.home-assistant.io/t/mqtt-publish-an-entity-state/286907/2
There are certainly other options. Possibly also PyScript or AppDaemon.
